packages/files/directory.pony

```diff
@@ -1,12 +1,12 @@
 use "time"
 
-use @ponyint_o_rdonly[I32]()
-use @ponyint_o_rdwr[I32]()
-use @ponyint_o_creat[I32]()
-use @ponyint_o_trunc[I32]()
-use @ponyint_o_directory[I32]()
-use @ponyint_o_cloexec[I32]()
-use @ponyint_at_removedir[I32]()
+use @pony_o_rdonly[I32]()
+use @pony_o_rdwr[I32]()
+use @pony_o_creat[I32]()
+use @pony_o_trunc[I32]()
+use @pony_o_directory[I32]()
+use @pony_o_cloexec[I32]()
+use @pony_at_removedir[I32]()
```

Comment:
Just curious - why renaming all of these supporting C functions to be "public" instead of internal (`pony_` instead of `ponyint_`)?